tests_e2e/tests/scripts/check_agent_update_from_log.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env pypy3
```

Comment:
We are copying all the test scripts to the same directory on the test VMs, and as we add more scripts, it's getting difficult to figure out that all those scripts are about. We should use a naming convention to facilitate this. Maybe add the test name as prefix for scripts that are specific to a test (e.g. agent_update-check_update.py)?